newIDE/app/src/Export/ExportDialog/ExportLauncher.js

```diff
@@ -121,7 +121,9 @@ export default class ExportLauncher extends Component<Props, State> {
       builds: [this.state.build],
       onBuildUpdated: (build: Build) => {
```

Comment:
This function is called every time a build is polled to check its status. When the status turns to 'done' or 'error', then the poll stops (after the last call)